packages/grafana-ui/src/components/Table/Table.tsx

```diff
@@ -195,8 +195,9 @@ export const Table: FC<Props> = memo((props: Props) => {
           {!noHeader && (
             <div>
               {headerGroups.map((headerGroup: HeaderGroup) => {
+                const { key, ...headerGroupProps } = headerGroup.getHeaderGroupProps();
                 return (
-                  <div className={tableStyles.thead} {...headerGroup.getHeaderGroupProps()}>
+                  <div className={tableStyles.thead} {...headerGroupProps} key={key}>
```

Comment:
Hmm, this is weird, was it giving a warning that `key` isn't explicitly provided?

Reply:
yes